models/DataObject/Objectbrick/Definition.php

```diff
@@ -263,7 +263,7 @@ public function save($saveDefinitionFile = true)
             foreach ($this->getFieldDefinitions() as $key => $def) {
 
                 /**
-                 * @var $def DataObject\ClassDefinition\Data
+                 * @var DataObject\ClassDefinition\Data $def
```

Comment:
single line and move down please

Reply:
I removed it. I set a return Type to getFieldDefinitions()